src/client/cli/cmd/launch.cpp

```diff
@@ -125,32 +125,16 @@ mp::ReturnCode cmd::Launch::run(mp::ArgParser* parser)
         return parser->returnCodeFrom(ret);
     }
 
-    request.set_time_zone(QTimeZone::systemTimeZoneId().toStdString());
-
     auto ret = request_launch(parser);
     if (ret == ReturnCode::Ok && request.instance_name() == petenv_name.toStdString())
     {
-        if (!MP_SETTINGS.get_as<bool>(mounts_key))
-        {
-            cout << fmt::format("Skipping '{}' mount due to disabled mounts feature\n", mp::home_automount_dir);
-        }
-        else
+        std::string mounts_value;
+        if (std::tie(ret, mounts_value) = request_mounts_setting_from_daemon(parser); ret == ReturnCode::Ok)
```

Comment:
This is really cool!  I never knew that could be done :smile:

Reply:
Yeah, it was used mostly before structured bindings, the downside is that the variables need to be already declared :smiley: